### PR TITLE
feature: add validate function

### PR DESCRIPTION
**Pull request by yamii**

### CODE REVIEW

**Comment by yamii** [comment]
don't merge yet there's still a question about the 
Error.UnkownSchemaError and ValidationError

**Comment by openam** [comment]
Is the plan to include this in [lapin](https://github.com/sinet/lapin)?

Then when writing tests we could just `require( '@sinet/validation' ).validate( message, handlerFile.config.validate ).then().catch()`

File: lib/validate.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+var Joi     = require( 'joi' );
+
+/*
+ * Validate using Hapi JS Joi Validate
+ * @params message {Object}
+ * 				- object message/value to be validated ( required )
+ * @params schema {Object}
+ * 				- object or joi object schema for validation ( required )
+ * @params options {Object}
+ * 			- addition joi options ( optional)
```
Comment (openam) [inline lib/validate.js:13]: align

File: lib/validate.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * 			refer here https://github.com/hapijs/joi/blob/v9.0.0-2/API.md#validatevalue-schema-options-callback )
+ *
+ * 	@return returns a Promise object
```
Comment (openam) [inline lib/validate.js:16]: align

**Comment by yamii** [comment]
@openam i'm closing this PR I also realized it's not good for lapin to use this package since the focus of this package is the common validation schemas for sinet. We'll figure out in the future for our common validation method

**Comment by openam** [comment]
Should we just add one thing here https://github.com/sinet/lapin/blob/master/index.js#L31 that exposes `validate` so that in testing services we can just require lapin, and execute validate on it?

**Pull request closed without merging**